app/models/incident.rb

```diff
@@ -94,6 +94,20 @@ def claim_for(user)
     save! validate: false
   end
 
+  def export_to_claims!
+    return unless completed?
+    begin
+      ci = ClaimsIncident.create claims_fields table: :incident
+      update claims_id: ci.UID
+      ClaimsDriversReport.create claims_fields table: :drivers_report
+    rescue ActiveRecord::StatementInvalid => e
+      puts e.cause and return false
+      # TODO: report failure to the user, and report error to programmers
```

Comment:
Could handle nerd-facing error reporting now, rather than putting it in with user-facing error reporting. Open to opinions.

Reply:
> nerd facing
😆

Reply:
Yeah, _until_ you take care of this TODO, I'm not sure you even need to `rescue` here. I think it's OK to wait to implement the error handling since this PR also doesn't include any way other than the model method to perform this action.